### DISCUSSION TITLE
Unifi Connect

### DISCUSSION THREAD

**Discussion by TwoTonik** [briis/unifiprotect#371] [General]
Anyone looking at Unifi Connect and hoping to use it as an HA Dashboard? I realize it is not part of Protect but you all have done such a good job on the HA/Protect Add-On I thought I would solicit your thoughts.

**Comment by AngellusMortis** [comment]
I already plan on making a separate package/integration for Connect. I have a UDIM switch already with a LED Panel in the mail. Just have to wait for those damn displays to go on sale.
